docs/pages/examples/react/player.en-US.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: 'Player'
+description: 'Learn how to use the livepeer.js media player'
+---
+
+import { DemoPlayer } from '@components/examples';
+
+# Player
+
+The example below shows our drop-in player for easily viewing media content! The example below uses the
+[`Player`](/react/Player) to view a video asset, with some custom styles to demonstrate what's possible with
+the player.
+
+<br />
+
+<DemoPlayer />
+
+## Step 1: Configuring Providers
+
+First, we create a new livepeer.js client with the Studio provider and a [CORS-protected API key](/react/providers/studio#apikey),
+as well as a theme to use for all livepeer.js React components.
+
+```tsx
+import {
+  LivepeerConfig,
+  ThemeConfig,
+  createReactClient,
+  studioProvider,
+} from '@livepeer/react';
+import * as React from 'react';
+
+const livepeerClient = createReactClient({
+  provider: studioProvider({
+    apiKey: process.env.NEXT_PUBLIC_STUDIO_API_KEY,
+  }),
+});
+
+const theme: ThemeConfig = {
+  colors: {
+    accent: 'rgb(0, 145, 255)',
+    containerBorderColor: 'rgba(0, 145, 255, 0.9)',
+  },
+  fonts: {
+    display: 'Inter',
+  },
+};
+
+function App() {
+  return (
+    <LivepeerConfig client={livepeerClient} theme={theme}>
```

Comment:
Do we think global app styles are necessary? Is the thinking here we might introduce more media UI elements to the library down the line that should inherit these global set of styles?

Reply:
Yeah, the reason for this is there will be multiple video elements on an app, so this would style *all* of them instead of each individually

Reply:
Is there a way of still configuring/override the theme for a specific player tho? I think it might be a use case to have players with different look&feel across the page as well.

Reply:
Yep! You can customize it [like this](https://livepeerjs-9assklo8k-livepeer.vercel.app/react/Player#theme).